client/src/store/new-request-module.ts

```diff
@@ -765,7 +785,7 @@ export class NewRequestModule extends VuexModule {
               territory, country or federal jurisdiction so that you may also conduct business here in BC.`
     },
     {
-      text: 'Relocate into BC a',
+      text: 'Relocate into a',
```

Comment:
Does this make sense?
"Relocate into a Canadian ... " ?

Reply:
This is what was demo'd to staff
The only location option is 'BC'. So it will read
'Relocate into a ..'   'BC'.. 'Bc entityType'..